types/publish.mjs

```diff
@@ -109,8 +109,7 @@ async function main() {
 
   // The majority of those items should be excluded entirely, but in some cases
   // we still need to post-process them.
-  let excludes = remappedLocationExcludes
-    .concat(sideEffectExcludes);
+  let excludes = remappedLocationExcludes.concat(sideEffectExcludes);
```

Comment:
This seems unrelated, it doesn't hurt but I'd revert it just to keep the diff minimal. I did notice we don't really have prettier set up here, which we can add in a different PR if we want.

Reply:
yeah, it's annoying to have an editor that enables prettier when a `.prettierrc.js` is detected, but some files haven't been formatted with it. :(

Reply:
updated